content/en/docs/TrueNAS/tn-aws.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "TrueNAS on Amazon Web Services (AWS)"
+linkTitle: "TrueNAS on AWS"
+description: "How to create a TrueNAS image and use it with Amazon Web Services"
+---
+
+{{% pageinfo version="FreeNAS 11.2" %}}
+{{% /pageinfo %}}
+
+## Process Summary
+
+* Requirements
+  * FreeBSD system
+  * AWS Account
+  * S3 Bucket
+  * User with permissions for EC2
+    * Download the user key to the local working directory
+  * Install bhyve and bsdec2-image-upload
+* Create TrueNAS image file
+  * Download TrueNAS .iso
+  * Create blank image file
+  * Load virtualization module
+  * Create tap and bridge interface
+  * Load image and iso into bhyve
+  * Install TrueNAS
+  * Reload image into bhyve
+* Configure image networking
+  * Reset network interface
+  * Create DHCP interface `xn0`
+  * Reset configuration to default
+  * Power off device
+* Upload image to EC2
+  * description and region name are required
```

Comment:
```suggestion
  * Description and region name are required
```

Reply:
Its the only line not capitalized, so it looks off.  If it should be lower case then ignore this suggestion